content/docs/command-reference/plots/diff.md

```diff
@@ -105,20 +105,32 @@ file:///Users/dmitry/src/plots/logs.html
 Compare two specific versions (commit hashes, tags, or branches):
 
 ```dvc
-$ dvc plots diff HEAD 0135527 --targets logs.csv
+$ dvc plots diff --targets logs.csv -- HEAD 0135527
 file:///Users/usr/src/plots/logs.csv.html
 ```
 
 ![](/img/plots_diff.svg)
 
-> Alternatively, you can also run above statement as:
+> Please note that when you're specifying `--targets` options with multiple
+> `revisions`:
+>
+> ```dvc
+> $ dvc plots diff --targets t1.json t2.csv HEAD v1 v2
+> ```
+>
+> This statement will show an error because argument parser confuses `revisions`
+> as arguments for `--targets` option. To avoid this error use `--` after all
+> the arguments for `--targets`:
 >
 > ```dvc
-> $ dvc plots diff --targets logs.csv -- HEAD 0135527
+> $ dvc plots diff --targets t1.json t2.csv -- HEAD v1 v2
 > ```
 >
-> When you're specifying multiple revisions after `--targets`, use `--` so that
-> argument parser doesn't confuse them as options for `--targets` argument.
+> Alternatively, you can also run above statement as:
+>
+> ```dvc
+> $ dvc plots diff HEAD v1 v2 --targets t1.json t2.csv
+> ```
```

Comment:
@jorgeorpinel is this part needed in `--target` option. I think rest of the part can be moved up as it and here instead we can use example in line 108 itself -> `$ dvc plots diff --targets logs.csv -- HEAD 0135527`. Along with one line note reminding to use `--`. 
Thoughts?

Reply:
I'm not sure, was this part of the example before? What about just use this one for the example (no --) and remove the note? Just try to summarize everything as much as possible, make your change, and we'll review that again. Thanks